frontend/src/components/activity/content/ColumnLayout.vue

```diff
@@ -1,38 +1,54 @@
 <template>
-  <v-row v-if="!contentNode.loading" no-gutters>
-    <resizable-column
-      v-for="(_, slot) in columns"
-      :key="slot"
-      :parent-content-node="contentNode"
-      :layout-mode="layoutMode"
-      :width-left="relativeColumnWidths[slot][0]"
-      :width="relativeColumnWidths[slot][1]"
-      :width-right="relativeColumnWidths[slot][2]"
-      :num-columns="numColumns"
-      :last="slot === lastColumn"
-      :min-width="minWidth(slot)"
-      :max-width="maxWidth(slot)"
-      :color="color"
-      :show-header="!isRoot"
-      @resizing="(newWidth) => resizeColumn(slot, newWidth)"
-      @resize-stop="saveColumnWidths"
-    >
-      <draggable-content-nodes
-        :slot-name="slot"
-        :layout-mode="layoutMode"
-        :parent-content-node="contentNode"
-        :disabled="disabled"
-      />
-
-      <template #menu>
+  <ContentLayout
+    v-resizeobserver.debounce="onResize"
+    class="ec-column-layout"
+    :class="{ 'ec-column-layout--layout-mode my-2': !isRoot && layoutMode }"
+    :is-root="isRoot"
+    :layout-mode="layoutMode"
+  >
```

Comment:
It will get reused in flex layout. I agree that the design is not consistent with Material Design and that it could be improved.
I opted for this design, as it is clearly different from the non-layout content types while still being compact. If you have a good idea how we could design this, I would like your input in a followup issue.